README.md

```diff
@@ -1,3 +1,79 @@
+# openSUSE Developer Mentoring Program
 
-Nothing to see here for now, move [along](https://www.opensuse.org/)!
+openSUSE is a community project to promote the use of Linux everywhere! We believe that the best results come from people collaborating and having fun. Our approach is to centralize a collection of project ideas for programs like Google Summer of Code, Rails Girls Summer of Code or Outreachy. 
+
+## Mentor
+### What makes a good mentor?
+We carefully picked some guidelines for mentoring! We strongly recommend our mentors to read them for more information about becoming a mentor: [Guidelines by FLOSS manuals](http://en.flossmanuals.net/GSoCMentoring/), [Mentoring.org](http://www.mentoring.org/downloads/mentoring_413.pdf), [Gnome mentoring guidelines](https://people.gnome.org/~federico/docs/summer-of-code-mentoring-howto/), [KDE mentoring guidelines](https://community.kde.org/Mentoring). 
+
+Furthermore, these are some skills and values we look for:
+* Time, time and time! Don't underestimate that mentoring is a time consuming task.
+* Being a FOSS evangelist and already part of an openSUSE project!
+* Have a clear vision of your mentoring project!
+* Having strong organization skills because you have to manage your Mentee and your daily life.
+
+### Adding a mentoring project
+If you want to add a mentoring project, please follow this workflow:
+
+1. Is your openSUSE project already listed on our projects site? If not, please contact first our [team]() and ask to add your project. Let's create awareness about our past, present and future projects!
+
+2. Go to our GitHub repository and add a [new issue](https://github.com/openSUSE/mentoring/issues) for your project. Please use this template:
+
+  **Project Title:** Project title, short enough to catch attention
+  
+  **Description:** General information about the project, avoid one Liners, the description should be as detailed as possible.
+  
+  **Deliverable:** Expectations from the student at the end of the project
+  
+  **Mentor:** Who is the mentor? Who is the Co-Mentor? Also please assign the issue to the mentor!
+  
+  **Skills:** Which skills are needed? Programming languages, frameworks, concepts etc.  
+  
+  **Skill Level:** Easy, Medium, Hard
+  
+  **Get started:** Tasks that mentors may want to suggest students so that they can start contributing to the code base (e.g. junior jobs, low hanging fruits, discussion on the mailing list)
+
+3. Label the issue with your projects name (e.g. Machinery) and assign it to yourself. It will now magically appear on our [project site]()!
+
+4. Wait till a Mentee gets in touch with you! 
+
+## Mentee
+
+You want to start working on a mentoring project? Awesome! Just pick one of the proposed project from our issue list and follow the get started instruction. Please also write a short comment in the issue that you would like to work on this project.
+
+### Google Summer of Code
```

Comment:
Maybe the GSoC instructions should better be on the web page.

We could add a generic description in the README saying something like "Google and others are running mentoring programs such as the Google Summer of Code. You can find instructions how to work on openSUSE projects there on the [mentoring web site](appropriate link). But you can always get involved if you find a project interesting. Just get in contact with the mentor by commenting on the GitHub issue of the project."